pyproject.toml

```diff
@@ -35,11 +35,13 @@ dependencies = [
     "click-plugins>=1.0",
     "cligj>=0.5",
     'importlib-metadata;python_version<"3.10"',
+    "pyparsing",
+    "shapely",
 ]
 
 [project.optional-dependencies]
 all = ["fiona[calc,s3,test]"]
-calc = ["shapely"]
+calc = ["pyparsing", "shapely"]
```

Comment:
This line can be removed since these are project dependencies (doing `pip install fiona[calc]` would just show a warning).

Reply:
@mwtoews making them core project requirements in this PR was me being sloppy. I've now moved them properly to the "calc" extras set and followed up in code and docs. I imagine there are still some few folks who want to continue to use fiona without shapely.